tests/unit/lms/views/content_item_selection_test.py

```diff
@@ -8,16 +8,6 @@
 
 
 class TestContentItemSelection:
-    def test_it_sets_the_authUrl_javascript_config_setting(
-        self, context, pyramid_request
-    ):
-        content_item_selection(context, pyramid_request)
-
-        assert (
-            context.js_config.config["authUrl"]
-            == "http://example.com/TEST_AUTHORIZE_URL"
-        )
-
```

Comment:
:+1: Ditto here reducing duplication. Let's not do the same thing _very_ slightly different.